Resources/Prototypes/Entities/Clothing/Hands/gloves.yml

```
@@ -354,6 +354,7 @@
   - type: MeleeWeapon
     autoAttack: true
     attackRate: 4
+    heavyStaminaCost: 2
     damage:
```

```suggestion
    attackRate: 4
    heavyStaminaCost: 2
    heavyDamageBaseModifier: 1.05
    maxTargets: 1
    damage:
```
This would give them only a 5% damage boost, in exchange for each 2 points of stamina spent on a power attack. This way, they aren't a DPS monster weapon that let's you take down whole crowds of people, but are instead a weapon that you can annihilate someone in a 1v1 with.